### ISSUE TITLE
No clone icon for organization items (Desktop app)

### ISSUE THREAD

**Issue by bwbug** [bitwarden/clients#12163]
### Steps To Reproduce

1. Set up a free organization, give user "Can Manage" access to a collection, and create at least one item in that collection.
2. Have the user (or the organization admin) log in to Desktop app.
3. Filter by collection.
4. Select an item for viewing.

### Expected Result

It is expected that a "Clone" icon should be available next to the "Edit" icon:

![image](https://github.com/user-attachments/assets/e9cfee0e-c77e-4eb8-acbb-257a12e47d3a)

### Actual Result

There is no "Clone" icon:

![image](https://github.com/user-attachments/assets/77b26a53-e3d8-4592-8666-cfc2b41582fe)

### Additional Context

The same lack of cloning functionality for organization items is observed in the browser extension (production version 2024.11.2), although the UI is different.

Cloning can still be done in the Web Vault, but it is no longer possible to change the ownership of cloned items (will file a separate bug report).

This constellation of bugs makes it impossible to copy (or move) an organization item into an individual vault, unless one uses the beta release of the browser extension.

### Operating System

Windows

### Operating System Version

Windows 11 23H2

### Installation method

Direct Download (from bitwarden.com)

### Build Version

Version 2024.11.2, Shell 32.1.1, Renderer 128.0.6613.137, Node 20.17.0, Architecture x64

### Issue Tracking Info

- [X] I understand that work is tracked outside of GitHub. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by rmcdowell-bitwarden** [comment]
Thank you for your report! 

I was able to reproduce this issue, and I have flagged this to our engineering team.

If you wish to add any further information/screenshots/recordings etc., please feel free to do so at any time - our engineering team will be happy to review these.

Thanks once again!

**Comment by alex-hammer11** [comment]
@rmcdowell-bitwarden After doing some digging, it seems deliberate in the sense that the if-statement showing the clone button checks if there isn't any organizationId in the cipher. I'm just a community developer so I can't tell if this should be the case or not. I'll include an image with code where line 662 is the one stating that there must not be an organizationId for the button to show, but I'll be happy to make any changes and open a pull request if needed 😄

![Screenshot 2024-12-09 at 11 25 45](https://github.com/user-attachments/assets/867753ef-058e-4aca-a354-5fb41e729fbb)

**Comment by closebot-bw** [comment]
⚠️ **Stale Issue Notice**

This issue has been automatically marked as stale due to inactivity. It will be closed in **2 weeks** (October 15, 2025) if no further activity occurs.

If this issue is still relevant and you would like to keep it open, please:
- Comment on this issue to show continued interest
- Provide any additional information or updates
- Confirm that the issue still exists in the latest version

Thank you for your contribution to this project! 🙏

**Comment by bwbug** [comment]
The issue is still present in version 2025.9.0 of the Desktop app.

Some of the information in the "Additional Context" section [above](https://github.com/bitwarden/clients/issues/12163#issue-2696802224) is outdated.   Cloning of organization items (for which the user has permissions to "Manage Collection") is possible in the Web Vault and in the browser extension, but not in the Desktop app.

**Comment by marcinBB8** [comment]
I have same problem. it is very annoying when need to add more similar items (just name ans pass is change, but have many other custom fields, that are same

**Comment by Rhahkeem** [comment]
So i'm currently seeing this issue in the Web Vauly as well as the browser extenstion. the clone option is just not there

**Comment by pamperer562580892423** [comment]
@Rhahkeem 

> So i'm currently seeing this issue in the Web Vauly as well as the browser extenstion. the clone option is just not there

I just checked both again - the clone option is in the item's three-dot-menu in the web vault and in the browser extension.

If it's not there for you: are you self-hosting? If yes, which server version?

**Comment by Rhahkeem** [comment]
this was my bad. on the web vault i was logged in as myself but i had to click "admin" to "login" and manage the console. I don't see it in the browser extension though. no i'm not self hosting. 

Extension Version
2026.2.0

**Comment by bwbug** [comment]
> I don't see it in the browser extension though.

Click on the "More Options" menu icon (:one: in screenshot below), and then select "Clone" (:two: in screenshot):

<img width="394" height="614" alt="Image" src="https://github.com/user-attachments/assets/c127eb75-007d-4482-afc3-0ad6e83e9fe9" />

**Comment by pamperer562580892423** [comment]
> I don't see it in the browser extension though. no i'm not self hosting.

@Rhahkeem Did you also make sure that the respective user has (at least) "Can manage" access to that collection the item is located in?

**Comment by daviddean-x** [comment]
Same here on macos and windows 
Permissions are set to manage to the collection, but there is no way to clone from desktop client,  are there any updates?

**Comment by daviddean-x** [comment]
@rmcdowell-bitwarden 
Will this be readded in near future? It would be helpful to create template items which employees can clone instead to manually create each time lot of custom files. Also with the clone feature we can make sure that each custom field has the correct naming :( 

Best regards 
David

**Comment by asentris** [comment]
This is also a bug in the android client.

**Comment by pamperer562580892423** [comment]
> ### Expected Result
> It is expected that a "Clone" icon should be available next to the "Edit" icon:
> 
> ![image](https://github.com/user-attachments/assets/e9cfee0e-c77e-4eb8-acbb-257a12e47d3a)
> 
> ### Actual Result
> There is no "Clone" icon:
> 
> ![image](https://github.com/user-attachments/assets/77b26a53-e3d8-4592-8666-cfc2b41582fe)

@bwbug Am I doing something wrong - or is that clone button/icon now also gone for "My vault" items (i.e. non-organizational items) in the "Edit item" dialog - in the desktop app, web vault and browser extension? Can you confirm that?

**Comment by bwbug** [comment]
@pamperer562580892423 The clone option is evidently now available in the kebab menu ("Options") for both individual and organizational items, just like in the browser extension.   If you confirm on your end, I will close this Issue as completed.

**Comment by pamperer562580892423** [comment]
@bwbug Yes, I can confirm this for the desktop app and web vault. And an improvement to have the clone option now for all items, and all in the same place.

For sure another "issue", but I fail to see now why the "options" (kebap menu) are scattered. Or what is the logic of having "Archive" and "Delete" in the "View item" windows - but not "Favorite", "Clone" and "Assign to collections".

And in "Edit item", one can "Delete" and "(Un-)Favorite", but not "(Un-)Archive", "Clone" or "Assign to collections".

It really seems to be the idea for users to memorize action locations. 🤪

**Comment by bwbug** [comment]
I agree that UI design is (still) not a forté for Bitwarden, but those are battles that we will have to fight elsewhere.  Closing this issue as implemented.
